DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -248,7 +248,8 @@
              newObj->setPflowIsolationVariables(pfIsoVar);]]>
   </ioread>
 
-  <class name="pat::Jet"  ClassVersion="17">
+  <class name="pat::Jet"  ClassVersion="18">
+   <version ClassVersion="18" checksum="1291486363"/>
```

Comment:
I think you also need to introduce a read rule as follows below
```xml
  <ioread sourceClass = "pat::Jet" version="[1-17]" targetClass="pat::Jet" source="" target="jerFactorValid_">
    <![CDATA[jerFactorValid_ = false;]]>
  </ioread>
```
to make sure this is correctly initialized when reading old versions of this class

Reply:
done